packages/headless/src/features/sort-criteria/criteria.ts

```diff
@@ -1,22 +1,58 @@
 import {isArray} from '@coveo/bueno';
 
+/**
+ * The available sort order
+ */
 export enum SortOrder {
   Ascending = 'ascending',
   Descending = 'descending',
 }
 
+/**
+ * The available criteria that can be used to sort query results
+ */
 export enum SortBy {
+  /**
+   * Use standard index ranking factors (adjacency, TDIDF) and custom ranking expressions (QREs and QRFs) to compute a ranking score for each query result item, and sort the query results by descending score value
+   */
   Relevancy = 'relevancy',
+  /**
+   * Use only custom ranking expressions (QREs and QRFs) to compute a ranking score for each query result item, and sort the query results by descending score value.
+   */
   QRE = 'qre',
+  /**
+   * date ascending/date descending: Use the date field to sort the query results. This field typically contains the last modification date of each item.
```

Comment:
On line 28, you use `@[field]`. Should this be `@[date]`?  or vice versa?

Reply:
It's different in this case. 
`@[fieldname] ascending/descending` would be the expression sent to the API to sort by field
`date ascending/descending` would be the expression sent to the API to sort by date (no leading `@` character).